addons/hr_work_entry_contract/wizard/hr_work_entry_regeneration_wizard.py

```diff
@@ -111,3 +111,15 @@ def regenerate_work_entries(self):
         write_vals = {field: False for field in self._work_entry_fields_to_nullify()}
         work_entries.write(write_vals)
         self.employee_ids.generate_work_entries(date_from, date_to, True)
+
+        return {
+            'type': 'ir.actions.act_window',
+            'res_model': 'hr.work.entry',
+            'view_mode': 'gantt,form',
+            'name': _('Work Entries'),
+            'domain': [
+                ('employee_id', 'in', self.employee_ids.ids),
+                ('date_stop', '>=', date_from),
+                ('date_start', '<=', date_to)
+            ],
+        }
```

Comment:
This is not really a workable solution since the action through which we arrive to this menu can be different. For example in community there is action_open_work_entries, so when we try to trigger this action (which has a gantt view) it will crash in community. Ideally, the wizard is simply closed and the previous view reloaded, which is what was already happening if you got to the menu through action_open_work_entries (the employee smartbutton) so it's not really clear to me why it's not happening (anymore?) when we got to this menu through the payroll menu button.